Dockerfile

```
@@ -2,21 +2,16 @@ FROM python:3.9-slim as build
 WORKDIR /wheel
 COPY . .
 RUN apt update && apt install -y build-essential git
-RUN cd /tmp && \
-    git clone https://github.com/errbotio/err-backend-slackv3 slackv3
-RUN pip3 wheel --wheel-dir=/wheel . \
-    -r /tmp/slackv3/requirements.txt wheel \
-    errbot errbot[irc] errbot[XMPP] errbot[telegram] && \
-    cp /tmp/slackv3/requirements.txt /wheel/slackv3-requirements.txt
+RUN pip3 wheel --wheel-dir=/wheel \
+    wheel . .[irc] .[XMPP] .[telegram] errbot-backend-slackv3
 
 FROM python:3.9-slim as base
 COPY --from=build /wheel /wheel
 RUN apt update && \
     apt install -y git && \
     cd /wheel && \
-    pip3 -vv install --no-cache-dir --no-index --find-links /wheel . \
-    -r /wheel/slackv3-requirements.txt \
-    errbot errbot[irc] errbot[XMPP] errbot[telegram] && \
+    pip3 -vv install --no-cache-dir --no-index --find-links /wheel \
+    . .[irc] .[XMPP] .[telegram] errbot-backend-slackv3 && \
```

You probably want to keep the package name `errbot` for the installation. Using the relative `.` makes sense for building the package automatically named by `setup.py`, but wouldn't give `pip` a concrete package name to install.
```suggestion
    errbot errbot[irc] errbot[XMPP] errbot[telegram] errbot-backend-slackv3 && \
```
You can also collapse the list of extras down which might help with legibility.
```suggestion
    errbot[irc,XMPP,telegram] errbot-backend-slackv3 && \
```

---

@e4r7hbug You make some valid points, however, the goal here is to create the master releases published to Dockerhub and I'd like to ensure we are just publishing the local tree changes.

I did, however, include your suggestion on simplifying the extras install.